src/server/oasisapi/files/serializers.py

```diff
@@ -29,7 +35,8 @@ def validate(self, attrs):
         return super(RelatedFileSerializer, self).validate(attrs)
 
     def validate_file(self, value):
-        if self.content_types and value.content_type not in self.content_types:
+        mapped_content_type = CONTENT_TYPE_MAPPING.get(value.content_type, value.content_type)
+        if self.content_types and value.content_type not in self.content_types and mapped_content_type not in self.content_types:
```

Comment:
This could be reduced to `if self.content_types and mapped_content_type not in self.content_types:`.